client/src/views/PizzasView.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import { useQuery } from 'urql';
+import Pizza, { PizzaToppingData } from '../components/Pizza';
+import { PizzasQuery } from '../data/query';
+
+interface PizzaData {
+  id: number;
+  name: string;
+  price: number;
+  crust: string;
+  image: string;
+  pizza_toppings: PizzaToppingData[];
+}
+
+interface PizzasData {
+  pizzas: PizzaData[];
+}
+
+export default function PizzasView() {
+  const [{ fetching, data }] = useQuery<PizzasData>({ query: PizzasQuery });
+
+  return fetching ? null : (
```

Comment:
great chance to put loader

Reply:
The loading time is so fast that when I put a loader/skeleton it looks like a glitch on the screen... 

![Screen Shot 2022-11-10 at 18 23 43](https://user-images.githubusercontent.com/41146054/201150856-c2e19e91-1a06-4fb5-961a-a1cac0402179.png)

The options I can think about are leaving the page is it is, or using a loader and give it a timeout for a minimum display time of 1s, just so it doesn't look glitchy.

What do you think?